ext/storage_sources/local_store/local_store.c

```diff
@@ -630,9 +630,10 @@ local_location_handle(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
     location->iface.close = local_location_handle_close;
     *location_handlep = &location->iface;
 
-    if (0)
+    if (0) {
 err:
         (void)local_location_handle_close(&location->iface, session);
```

Comment:
Coverity doesn't seem to spot this cleanup and complains about leaked `location` and `location->bucket` memory in the error case.

The only weird thing about this line is the lack of braces (when we put the error-tag in here, we usually put braces). I'd hope that their parser is smart enough to figure this out but I figure we may as well give it a try and see if it goes away.